docs/connector-development/config-based/tutorial/6-testing.md

```diff
@@ -0,0 +1,99 @@
+# Step 5: Testing
+
+We should make sure the connector respects the Airbyte specifications before we start using it in production.
+This can be done by executing the Source-Acceptance Tests (SAT).
+
+These tests will assert the most basic functionalities work as expected and are configured in `acceptance-test-config`.
+
+Before running the tests, we'll create an invalid config to make sure the `check` operation fails if the credentials are wrong, and an abnormal state to verify the connector's behavior when running with an abnormal state.
+
+Update `integration_tests/invalid_config.json` with this content
+
+```
+{"access_key": "<invalid_key>", "start_date": "2022-07-21", "base": "USD"}
+```
+
+and `integration_tests/abnormal_state.json` with
+
+```
+{
+  "rates": {
+    "date": "2999-12-31"
+  }
+}
+
+```
+
+You can build the connector's docker image and run the acceptance tests by running the following commands:
+
+```
+docker build . -t airbyte/source-exchange-rates-tutorial:dev
+python -m pytest integration_tests -p integration_tests.acceptance
+```
+
+1 test should be failing
```

Comment:
there are currently 2 tests failing. the second one will be fixed by https://github.com/airbytehq/airbyte/pull/15067

Reply:
I confirm I have the following two failures:
```
 - airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/tests/test_core.py:183 TestConnection.test_check[inputs1]
- airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/tests/test_incremental.py:207 TestIncremental.test_state_with_abnormally_large
 ```